service/client.go

```diff
@@ -97,7 +97,7 @@ func (s *server) pushLog(ctx context.Context, id thread.ID, lg thread.LogInfo, p
 	client := pb.NewServiceClient(conn)
 	_, err = client.PushLog(cctx, lreq)
 	if err != nil {
-		log.Warnf("push log to %s failed: %s", pid.String(), err)
+		return fmt.Errorf("push log to %s failed: %s", pid.String(), err)
```

Comment:
This might fail the existing `AddFollower` tests... we'll see. I was seeing cases where the log was successfully pushed to the new follower, but the addresses were updated anyway, meaning that get records would probably fail forever. May we well error up front.

Reply:
Looks like all tests are still passing. Safe to move forward with this then?

Reply:
Nice 👍

Reply:
(updated comment above which was confusing)